data/python.gram

```diff
@@ -303,10 +303,10 @@ class Parser(Parser):
                 self._tokenizer.get_lines(list(range(start[0], end[0] + 1)))
             )
 
-        self._exception = SyntaxError(
-            message,
-            (self.filename, start[0], start[1], line)
-        )
+        args = (self.filename, start[0], start[1], line)
+        if sys.version_info >= (3, 10):
+            args += (stop[0], stop[1])
```

Comment:
I assume this is for printing the new ranged errors in Python 3.10, no?

Reply:
yes

Reply:
And because I realize when trying to add testing that it was missing.